lang/python/python-yaml/Makefile

```diff
@@ -8,36 +8,59 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=PyYAML
-PKG_VERSION:=3.11
+PKG_VERSION:=3.12
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://pyyaml.org/download/pyyaml/
-PKG_MD5SUM:=f50e08ef0fe55178479d3a618efe21db
+PKG_HASH:=592766c6303207a20efc445587778322d7f73b161bd994f227adaa341ba212ab
 
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=COPYING
 PKG_MAINTAINER:=Nicolas Thill <nico@openwrt.org>
 
-PKG_BUILD_DEPENDS:=python
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-$(PKG_NAME)-$(PKG_VERSION)
+PKG_UNPACK=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
 
 include $(INCLUDE_DIR)/package.mk
 $(call include_mk, python-package.mk)
+$(call include_mk, python3-package.mk)
 
-define Package/python-yaml
+define Package/python-yaml/Default
   SUBMENU:=Python
   SECTION:=lang
   CATEGORY:=Languages
-  TITLE:=YAML parser and emitter for Python
   URL:=http://pyyaml.org/wiki/PyYAML
-  DEPENDS:=+python +libyaml
+  DEPENDS:=+libyaml
+endef
+
+define Package/python-yaml
+$(call Package/python-yaml/Default)
+  TITLE:=YAML parser and emitter for Python
+  DEPENDS+=+PACKAGE_python-yaml:python-base
+  VARIANT:=python
 endef
 
-define Build/Compile
+define Package/python3-yaml
+$(call Package/python-yaml/Default)
+  TITLE:=YAML parser and emitter for Python3
+  DEPENDS+=+PACKAGE_python3-yaml:python3-base
```

Comment:
this needs to be `+PACKAGE_python3-yaml:python3-light` to run
seems that the `python3-base` does not package the `re` package [while the `python-base` package does 
and python-yaml seems to need it

i tried just to add the `re` module into the `python3-base` package, but it seems to require the `enum` module
```
root@LEDE:/# python3
Python 3.6.1 (default, Jul  9 2017, 19:44:34) 
[GCC 5.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import yaml
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/__init__.py", line 8, in <module>
  File "/loader.py", line 4, in <module>
  File "/reader.py", line 22, in <module>
  File "/re.py", line 122, in <module>
ModuleNotFoundError: No module named 'enum'
>>> 
```

I stopped after the `enum` module was required.
Even tho it's a bit inconsistent with the `python` variant, I'd say to use the `python3-light` here as a dependency.
It seems to work fine with that [ re & enum modules are in there ].